src/main/java/chess/domain/piece/Bishop.java

```diff
@@ -0,0 +1,49 @@
+package chess.domain.piece;
+
+import chess.domain.Position;
+import chess.domain.piece.character.Character;
+import chess.domain.piece.character.Kind;
+import chess.domain.piece.character.Team;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bishop extends Piece {
+    public Bishop(Team team) {
+        this(team, false);
+    }
+
+    private Bishop(Team team, boolean hasMoved) {
+        super(team, hasMoved);
+    }
+
+    @Override
+    public Piece move() {
+        if (hasMoved) {
+            return this;
+        }
+        return new Bishop(team, true);
+    }
+
+    @Override
+    public Character findCharacter() {
+        return Character.findCharacter(team, Kind.BISHOP);
+    }
+
+    @Override
+    protected List<Position> findBetweenPositions(Position position, int rowDifference, int columnDifference) {
+        int absoluteDifference = Math.abs(rowDifference);
+        int rowSign = rowDifference / absoluteDifference;
+        int columnSign = columnDifference / absoluteDifference;
+
+        List<Position> positions = new ArrayList<>();
+        for (int movement = MIN_MOVEMENT; movement < absoluteDifference; movement++) {
```

Comment:
```suggestion
        for (int i = MIN_MOVEMENT; i < absoluteDifference; i++) {
```
for문의 i를 건드리지 않는것을 추천드립니다
지금은 for문안의 로직이 단순해서 크게 헷갈리지 않지만
for문의 길이가 조금만 길어져도 어떤 값이 반복되는 값인지 한눈에 파악하기 어렵습니다!

Reply:
변수명을 이해하기 쉽도록 하기 위한 방법이였는데, 어떤 값이 반복되는 값인지 파악하기 어렵다는 단점이 있군요!

변수가 무엇인지 파악하는 것은 위의 min,max로 충분한건가요?